scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -77,6 +86,14 @@ namespace <%= namespace %> {
             });
         }
 
+        public void StartWebCheckout(CheckoutSuccessCallback success, CheckoutFailureCallback failure, CheckoutCancelCallback cancelled) {
```

Comment:
I like `CheckoutWithNativeWebView` over `CompleteNativeWebViewCheckout` mostly because we beginning of the checkout process.